### PR TITLE
DP-11301: feedback integration updates

### PR DESCRIPTION
**Pull request by tnathanjames**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)

## Description

This provides feedback form updates to be pulled into the feedback form changes in Drupal.

## Related Issue / Ticket

- [DP-11301](https://jira.mass.gov/browse/DP-11301)

## Steps to Test

- Checkout the mayflower branch
- Build locally
- Confirm that the patterns in Organisms->feedback match intended designs
- Testing in Drupal is also intended for approving this ticket.  See https://jira.mass.gov/browse/DP-11148 for more information on the Drupal testing.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

* 

#### @TODO

*

### CODE REVIEW

**Comment by tnathanjames** [comment]
Closing in favor of another PR.

File: assets/scss/00-base/_transitions.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Transitions.
+//
+// Standard transitions used within MF.
+
+$transition-slow: 600ms ease-in-out;
+$transition-normal: 400ms ease-in-out;
+$transition-fast: 200ms ease-in-out;
```
Comment (clair0917) [inline assets/scss/00-base/_transitions.scss]: I don't think these variables get used anywhere. We can probably remove this.

File: assets/scss/00-base/_variables.scss [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
-
-$l-max-content-width: 820px;
-$l-max-content-wrapper: 1240px;
-
-//letter-spacing for Texta
-$letter-spacing-large: .1em;
-$letter-spacing-medium: .05em;
-$letter-spacing-small: .02em;
-$letter-spacing-default: 0;
+// Change the Neat grid settings
```
Comment (samseide) [inline assets/scss/00-base/_variables.scss]: need a period at end of comment.

File: assets/scss/01-atoms/global/_elements.scss [hunk 3]
```diff
@@ -1,173 +1,173 @@
...
+
+//   // WebKit gets its native focus styles.
+
+//   @media (-webkit-min-device-pixel-ratio:0) {
+//     outline-style: auto;
+//   }
+// }
+
+// WARNING: avoid nesting of elements,
+// because it makes them harder to override
```
Comment (samseide) [inline assets/scss/01-atoms/global/_elements.scss]: add a period.

File: changelogs/DP-11135.txt [hunk 4]
```diff
@@ -0,0 +1,30 @@
+___DESCRIPTION___
```
Comment (samseide) [inline changelogs/DP-11135.txt:1]: This file seems to not be finished.

File: patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback-contact-requested.json [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    "labelText": "Text Area",
+    "hiddenLabel": true,
+    "required": false,
+    "maxlength": 1000,
+    "id": "negative-textarea",
+    "name": "negative-textarea"
+  },
+  "helperText": {
+    "inputId": "GUID2738572489",
+    "message": "Do not include sensetive information, such as Social Security or bank account numbers."
```
Comment (samseide) [inline patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback-contact-requested.json]: sensetive <- misspelling

File: patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback-success-thankyou.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+### Description
+This Pattern contains the markup for the feedback form success and thanky you panels
```
Comment (samseide) [inline patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback-success-thankyou.md]: thanky <- misspelling

File: patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    "labelText": "Text Area",
+    "hiddenLabel": true,
+    "required": false,
+    "maxlength": 1000,
+    "id": "negative-textarea",
+    "name": "negative-textarea"
+  },
+  "helperText": {
+    "inputId": "GUID2738572489",
+    "message": "Do not include sensetive information, such as Social Security or bank account numbers."
```
Comment (samseide) [inline patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback.json]: sensetive <- misspelling. Should be sensitive

**Review by samseide — COMMENTED** [review]
added comments

File: assets/scss/02-molecules/_radio-scale.scss [hunk 8]
```diff
@@ -0,0 +1,88 @@
+// Radio Scale
+//
+// Custom radio set that displays the otions in a block'd scale.
```
Comment (samseide) [inline assets/scss/02-molecules/_radio-scale.scss:3]: otions - options

**Review by samseide — APPROVED** [review]

**Pull request merged**